src/index.html

```diff
@@ -6,7 +6,345 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page-catalog">
+    <header class="header">
+      <a class="header_logo" href="#">
+        <img
+          class="header_logo"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link nav__link--active is-active" href="#apple">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#phones">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#computers"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">
+              Гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#notepads">
+              Планшеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">
+              Фото
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#video">
+              Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog page-catalog__catalog">
+      <div class="catalog__product-card product-card"
+             data-qa="card">
+        <img
+              class="product-card__imac_img"
+              src="images/imac.jpeg"
+              alt="imac"
+        >
+
+        <h2 class="product-card__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+        <p class="product-card__code">Код товара: 195434</p>
+
+        <div class="product-card__rating">
+          <div class="product-card__stars stars">
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star"></div>
```

Comment:
don't hardcode active stars. Use logic written in the task layout_stars for stars activeness.